types/mixpanel-browser/index.d.ts

```diff
@@ -158,8 +162,8 @@ export interface Mixpanel {
     opt_in_tracking(options?: Partial<InTrackingOptions>): void;
     opt_out_tracking(options?: Partial<OutTrackingOptions>): void;
     push(item: PushItem): void;
-    register(props: Dict, days?: number): void;
-    register_once(props: Dict, default_value?: any, days?: number): void;
+    register(props: Dict, days_or_options?: number | Partial<RegisterOptions>): void;
```

Comment:
Since all the usages of `RegisterOptions` are used with Partial<`RegisterOptions>` mark persistent as optional in the interface instead of at usage site.

Reply:
I tried to align with the convention of this file. Interfaces are defined without optional keys on them and `Partial` is added when using this interface. Lines 158, 162, and 163 for example.